test/test_other.py

```diff
@@ -6901,7 +6901,8 @@ def test_browser_language_detection(self):
     # We support both "C" and "en_US" here since older versions of node do
     # not expose navigator.languages.
     lang = os.environ.get('LANG', 'C.UTF-8')
```

Comment:
Maybe rename `lang` to `expected_lang` or `host_lang` to make this more clear for future readers.